pkgs/development/ocaml-modules/sail/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, buildDunePackage
+
+, base64
+, omd
+, menhir
+, ott
+, linenoise
+, dune-site
+, pprint
+, makeWrapper
+, lem
+, z3
+, linksem
+, num
+}:
+
+buildDunePackage rec {
+  pname = "sail";
+  version = "0.15";
+
+  src = fetchFromGitHub {
+    owner = "rems-project";
+    repo = "sail";
+    rev = version;
+    hash = "sha256-eNdFOSzkniNvSCZeORRJ/IYAu+9P4HSouwmhd4BQLPk=";
+  };
+
+  duneVersion = "3";
+  minimalOCamlVersion = "4.08";
+
+  SAIL_DIR = "$out";
```

Comment:
Could this be moved to `makeFlags` (or equivalent for `buildDunePackage`, I know little about ocaml packaging.)